README.md

```diff
@@ -1 +1,191 @@
+
+**OR**
+
+##### b. Add an entry credit address
+
+Replace the secret key with your own secret key. It should begin with `Es`. Note: be careful not to change the formatting when copying over your secret key. Check the quotation marks.
+
+```shell script
+docker-compose exec walletd curl -X GET --data-binary '{"jsonrpc": "2.0", "id": 0, "method": "import-addresses", "params":{"addresses":[{"secret":"INSERT YOUR SECRET KEY HERE"}]}}' -H 'content-type:text/plain;' http://localhost:8089/v2
+```
+
+##### 9. Create a FCT address (if you don't already have one)
+
+If you do not already have a FCT address, you can create one in walletd.
+
+```shell script
+docker-compose exec walletd curl -X GET --data-binary '{"jsonrpc": "2.0", "id": 0, "method": "generate-factoid-address"}' -H 'content-type:text/plain;' http://localhost:8089/v2
+```
+
+Copy down the public address, you will need it later.
+
+#### 10. Fund your entry credit address
+
+The cheapest way to buy entry credits is to create them yourself using FCT. You can do that using several different Factom wallets, including [MyFactomWallet](https://myfactomwallet.com/#/) (web), [Enterprise Wallet](https://docs.factomprotocol.org/wallets/enterprise-wallet) (desktop) or walletd (cli), which we have running here.
+
+Alternatively, you can buy EC directly from [Factom Inc](https://shop.factom.com/) or [De Facto](https://ec.de-facto.pro/) who will deposit EC directly into your entry credit address.
+
+#### 11. Configure the PegNet config file
+
+Open the config file in a text editor.
+
+```
+nano config/pegnet.ini
+```
+
+Change the following items:
+
+-   `ECAddress` to the public EC address created or added to the wallet in step 5.
+-   `CoinbaseAddress` to your FCT address.
+-   `IdentityChain` to any alphanumeric string (no spaces!) that will help you identity your miner amongst the rest.
+
+Your miner needs to be able to get asset prices from an oracle. Complete the `[Oracle]` and `[OracleDataSources]` section of the config. The comments in the config file will guide you.
+
+## Start
+
+#### Starting the miners
+
+When you start PegNet, you need to pass the number of miners you want it to run as an environment variable. Each miner will require a single CPU core.
+
+Make sure you're in the pegnet-compose directory.
+
+##### Local Mining
+
+```
+MINERS=4 docker-compose up -d pegnet
+```
+
+
+##### Network Mining
+
+###### On coordinator machine:
+
+```shell script
+docker-compose up -d pegnet-netcoordinator
+```
+
+###### On miner machines:
+
+You will need to edit the following setting on `config/pegnet.ini` to the IP/Port of your coordinator:
+```ini
+MiningCoordinatorHost=localhost:7777
+```
+
+and then
+
+```shell script
+docker-compose up -d pegnet-netminer
+```
+
+
+#### Checking the logs
+
+Check the log output to make sure it is healthy. Mining will start once factomd begins to make progress, which may take a while.
```

Comment:
The way the rest of the tutorial is now phrased means that factomd should definitely be progressing. Therefore, we can probably get rid of this statement now. Just `Check the log output to make sure it is healthy.` would be fine, in my opinion.